### PR TITLE
add static directory option to storybook build script (GWY-13645)

### PR DESCRIPTION
**Pull request by agualis**
### What was a problem?

We forgot to add -s option when building storybook so the static was not found giving 404 errors in the deployed version of storybook

### How this PR fixes the problem?

Adding `-s ./public` to `build-storybook` script in` package.json` so that static contents are copied when building. 

### Check lists

- [ x] Readme file updated with actual description and example

### CODE REVIEW

**Comment by ZengJunyong** [comment]
So if this PR is merged to master then this ticket will be solved, let's have a try :-)

File: packages/recomponents/README.md [hunk 1]
```diff
@@ -53,6 +53,28 @@ Or bundled with webpack:
...
+    yarn global add http-server
+    npm install http-server -g
+```
+
+and run: 
+```bash
+    yarn serve-built-storybook
+    npm run serve-built-storybook
+```
 ## Licence
```
Comment (adamaltman) [inline packages/recomponents/README.md]:
```suggestion
## License
```
I know you didn't adjust this, but we follow US English and I just noticed this here.

Comment (agualis) [reply packages/recomponents/README.md]: 👍

**Pull request merged**